modules/docextract/lib/refextract_kbs.py

```diff
@@ -203,8 +201,6 @@ def institute_num_pattern_to_regex(pattern):
                             ('yy',   r'\d\d'),
                             ('s',    r'\s*'),
                             (r'/',   r'\/')]
-    # first, escape certain characters that could be sensitive to a regexp:
-    pattern = re_report_num_chars_to_escape.sub(r'\\\g<1>', pattern)
```

Comment:
can't really comment on the KB changes, they appear ok/plausible. but beyond the removal of the non-ascii escaping regex I also wonder if this was checked against overly broad matches. It's not apparent from the code